intera_interface/src/intera_interface/limb.py

```diff
@@ -197,6 +229,9 @@ def _on_endpoint_states(self, msg):
             ),
         }
 
+    def _on_tip_states(self, msg):
+        self._tip_states = deepcopy(msg)
```

Comment:
That's true. We might be able to avoid a deep copy, and just reassign the message to tip states